schemars_derive/src/attr/validation.rs

```diff
@@ -316,11 +316,7 @@ impl ValidationAttrs {
         let mut object_validation = Vec::new();
         let mut string_validation = Vec::new();
 
-        if let Some(length_min) = self
-            .length_min
-            .as_ref()
-            .or_else(|| self.length_equal.as_ref())
-        {
```

Comment:
Doesn't clippy usually want the `_else` case if you run a function call in the `or` case? Because the function is only executed if the closure is called. Don't have any objections to this, just did not expected this.

Reply:
Well, it might have been a recent change. I've been using clippy on nightly.